source/elements/oneMKL/source/domains/blas/iamax.rst

```diff
@@ -59,7 +55,8 @@ iamax (Buffer Version)
                   std::int64_t n,
                   sycl::buffer<T,1> &x,
                   std::int64_t incx,
-                  sycl::buffer<std::int64_t,1> &result)
+                  sycl::buffer<std::int64_t,1> &result,
+                  index_base base = index_base::zero)
```

Comment:
I think the standard is to add the namespace for the types (like onemkl::transpose transa in gemm)